test_agent/python/testagent.py

```diff
@@ -288,29 +299,43 @@ def handle_uri_validate_command(json_msg):
             ACTION_COMMANDS.VALIDATE_URI,
             received_test_id=json_msg["test_id"],
         )
+    else:
+        send_to_test_manager(
+            {
+                "result": "False",
+                "message": "Nonexistent UriValidator function",
+            },
+            CONSTANTS.VALIDATE_URI,
+            received_test_id=json_msg["test_id"],
+        )
+
 
 def handle_micro_serialize_uri_command(json_msg: Dict[str, Any]):
     uri: UUri = dict_to_proto(json_msg["data"], UUri())
     serialized_uuri: bytes = MicroUriSerializer().serialize(uri)
     # Use "iso-8859-1" to decode bytes -> str, so no UnicodeDecodeError if "utf-8" decode
     serialized_uuri_json_packed: str = serialized_uuri.decode("iso-8859-1")
     send_to_test_manager(
-        serialized_uuri_json_packed, 
-        ACTION_COMMANDS.MICRO_SERIALIZE_URI, 
-        received_test_id=json_msg["test_id"]
+        serialized_uuri_json_packed,
+        ACTION_COMMANDS.MICRO_SERIALIZE_URI,
+        received_test_id=json_msg["test_id"],
```

Comment:
is there any specific need to provide extra comma at the end?

Reply:
there's a formatter that made the recommendation

Reply:
Which formatter we are using? Isn't it should be PEP8?

Reply:
we are using "black formatter" in vscode